lib/config.js

```diff
@@ -59,49 +59,78 @@ function isFilePath(filePath) {
 }
 
 /**
- * Load and parse a JSON config object from a file.
+ * Check if item is an javascript object
+ * @param {*} item object to check for
+ * @returns {boolean} True if its an object
+ * @private
+ */
+function isObject(item) {
```

Comment:
Why not use https://www.npmjs.com/package/lodash.isplainobject instead? It does [a bit more](https://github.com/lodash/lodash/blob/master/lodash.js#L8289).